src/Catlab.jl

```diff
@@ -0,0 +1,74 @@
+# Catlab.jl 
```

Comment:
this is not something you should include in this PR, you could have it locally if you just wanna try it out.